README.md

```diff
@@ -77,8 +77,23 @@ func main() {
 ```
 
 Notes on examples above:
+
 - Migrator creates/manages a table named `migrations` to keep track of the applied versions. However, if want to customize the table name `migrator.TableName("my_migrations")` can be passed to `migrator.New` function as an additional option. 
 
+### Logging
+
+By default, migrator prints applying/applied migration info to stdout, it that's enough for you, you can skip this section.
```

Comment:
Typo and should be two sentences:  By default, migrator prints applying/applied migration info to stdout. `If` that's enough for you, you can skip this section.